web/.htaccess

```diff
@@ -0,0 +1,13 @@
+<IfModule mod_rewrite.c>
+        Options -MultiViews
+        RewriteEngine On
+        RewriteBase /
+        RewriteCond %{HTTPS} off
+        RewriteRule .* https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+        RewriteCond %{REQUEST_FILENAME} !-d
+        RewriteCond %{REQUEST_FILENAME} !-f
+        RewriteRule ^ index.php [QSA,L]
```

Comment:
These three lines are basically a replacement for [the 404 page](https://software.wikimedia.de/404). I think this is ok. What I wonder about is why "query string append" is done here? Are there any parameters index.php accepts?